src/main/java/setup/cardlists/MakeAllCardsList.java

```diff
@@ -15,37 +15,46 @@
 public class MakeAllCardsList {
 
 	//This code is meant to be run using some kind of scheduler.
+	private static boolean test = false;
 
-	public static void main(String[] args) throws Exception{
+	public static void main(String[] args) throws Exception {
 
-		final int TIMES_CHECKED = 168; //Hours in a week
-		//final int TIMES_CHECKED = 1; //For testing
+		final int TIMES_CHECKED = (test ? 1 : 168); // 168 hours per week
 
 		//Create a list of set abbreviations from the provided file, and check how many times the program has been run so far.
-		List<String> setAbbr; 
+		List<String> setAbbr;
 
-		if (!new File("MTGO_Set_Abbreviations").exists())
-		{
-			setAbbr = FileConverter.readToList(MakeAllCardsList.class.getClassLoader().getResource("MTGO_Set_Abbreviations"));
-		}
-		else {
-			setAbbr = FileConverter.readToList("MTGO_Set_Abbreviations");			
+		if (test) {
+			if (!new File("MTGO_Set_Abbreviations_test").exists())
+			{
+				setAbbr = FileConverter.readToList(MakeAllCardsList.class.getClassLoader().getResource("MTGO_Set_Abbreviations_test"));
+			}
+			else {
+				setAbbr = FileConverter.readToList("MTGO_Set_Abbreviations_test");
+			}
+		} else {
+			if (!new File("MTGO_Set_Abbreviations").exists())
+			{
+				setAbbr = FileConverter.readToList(MakeAllCardsList.class.getClassLoader().getResource("MTGO_Set_Abbreviations"));
+			}
+			else {
+				setAbbr = FileConverter.readToList("MTGO_Set_Abbreviations");
+			}
 		}
-		if (!new File("Count.txt").exists())
```

Comment:
Nit. If we're changing (or maybe "settling on") brace style then we should apply it throughout.